index.php

```diff
@@ -1,6 +1,8 @@
 <?php
 include('loklak.php');
-
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
```

Comment:
Can't we call this as `debug.php` and include_once if needed, we're cluttering both loklak.php and index.php with this, This can either be removed or put as a sample debug file.